### PR TITLE
2.0/feature navigation system/front end

### PR DESCRIPTION
**Pull request by maks100ful**

### CODE REVIEW

File: SportHub/src/pages/Admin/styles/style.module.scss
Comment (XashDM) [inline SportHub/src/pages/Admin/styles/style.module.scss:1]: Is this file empty why we need this

File: SportHub/src/pages/Admin/components/AdminPage.js
Comment (XashDM) [inline SportHub/src/pages/Admin/components/AdminPage.js:1]: Try not to commit unnecessary changes

File: SportHub/src/modules/NavigationAdminSystem/components/MenuItems/components/MenuItems.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import {useState,useTransition,useEffect,useRef,Suspense} from "react";
+import styles from "../styles/style.module.scss"
+import SubCategoryRequest from "../helpers/RequestsSubCategories";
+import TeamsRequest from "../helpers/RequestTeam";
+// import Dropdown from "./Dropdown";
```
Comment (XashDM) [inline SportHub/src/modules/NavigationAdminSystem/components/MenuItems/components/MenuItems.js:6]: remove

File: SportHub/src/modules/VerticalAdminMenu/components/VerticalAdminMenu.js [hunk 2]
```diff
@@ -19,10 +20,16 @@ export default function VerticalAdminMenu(){
                         <VerticalAdminMenuElement
                             name={verticalMenuElement}
                             hintText={hintText[index]}
-                            isActive={index === currentVerticalMenuElement}  />
+                            isActive={index === currentVerticalMenuElement} />
                     </div>
                 })}
             </div>
+            {currentVerticalMenuElement === 5 && (
+                <div className={styles.module_content_container}>
```
Comment (XashDM) [inline SportHub/src/modules/VerticalAdminMenu/components/VerticalAdminMenu.js:28]: Ask Roman how he is implementing this selection, or we can discuss it on Thursday

File: SportHub/src/modules/NavigationAdminSystem/components/MenuItems/components/MenuItems.js
Comment (XashDM) [inline SportHub/src/modules/NavigationAdminSystem/components/MenuItems/components/MenuItems.js:1]: Rename to NavigationItems to show the purpose of component -> menu is to generic name

File: SportHub/src/modules/NavigationAdminSystem/components/MenuItems/components/Dropdown/components/Dropdown.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// import MenuItems from "../../MenuItems";
+import React,{Suspense} from "react";
+import styles from "../styles/style.module.scss"
+import Xarrow from "react-xarrows";
+
+const MenuItems = React.lazy(() => import("../../MenuItems"))
+
+const Dropdown = ({parent,submenus,dropdown,depthLevel}) => {
```
Comment (XashDM) [inline SportHub/src/modules/NavigationAdminSystem/components/MenuItems/components/Dropdown/components/Dropdown.js:8]: What is responsibility of this component

File: SportHub.Database/Config.cs [hunk 4]
```diff
@@ -8,10 +8,10 @@ namespace SportHub.Database.Config
     class Config
     {
        public static string server = "localhost";
-       public static string uid = "user";
-       public static string password = "root";
-       public static string port = "5900";
-       public static string scriptFilePath = @"CreateDB.sql";
+       public static string uid = "root";
```
Comment (XashDM) [inline SportHub.Database/Config.cs:11]: Why you need to change this?

**Review by Oleh-Kril — APPROVED** [review]
Fixes of main problems are correct
